Makefile.ac

```diff
@@ -187,6 +188,7 @@ flight_plan_ac_h : $(FLIGHT_PLAN_H) $(FLIGHT_PLAN_XML)
 
 makefile_ac: $(MAKEFILE_AC)
 
+ifeq ($(GENERATE_KEYS),1)
```

Comment:
Can you please elaborate? Even better if you could outline the solution, I am not sure if I understand enough makefile magic.

Reply:
see https://github.com/paparazzi/paparazzi/tree/gec_module_makefile
if it fails afterward because no keys are available, maybe we can provide dummy keys for compilation purpose

Reply:
Great, thanks! The easy way is to remove the airframe with secure link from tests, but that is not a sustainable solution. Can we add Rust on Travis?

Reply:
around here I guess: https://github.com/paparazzi/paparazzi/blob/master/.travis.yml#L12

Reply:
Fixed